### PR TITLE
add 17 nov 2015 meeting notes (first pass)

### PR DESCRIPTION
**Pull request by ashleygwilliams**
because we had limited people, i volunteered to scribe top level issues.
full notes will be made after viewing the recording.

### CODE REVIEW

**Comment by sup** [comment]
participants? ([like this?](https://github.com/nodejs/LTS/blob/master/doc/meetings/2015-06-29.md))

File: meetings/2015-11-17.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ - CoC 1.0
+
+## Notes
+
+### Scope/mission of the WG
+- short term have it's own autonomy
+- long term extend to the larger node community
+- moderate ourselves,make recommendations for larger Node project
+- we don't want to be involved enforcement
+- legislative, judicial
```
Comment (aredridel) [inline meetings/2015-11-17.md]: Not sure what this means

**Comment by ashleygwilliams** [comment]
good call @sup .. any chance you wanna make a PR for that? i am over extended rn

File: meetings/2015-11-17.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### Balancing Privacy and Transparency
+- Overwhelming agreement that private channels for incident reporting and discussion
+- All decisions and outcomes will be publicly announced, rationalized and discussed
+- Not "coming down from on high"
+- Need a way to cut the noise and focus discussion
+- Be clear and structured in regards to the feedback we want on a given issue
+- Make private discussions public, read-only
```
Comment (aredridel) [inline meetings/2015-11-17.md]: When? in what cases?

**Comment by ashleygwilliams** [comment]
you can just PR to this branch (`2015-11-17`) and i'll merge

**Comment by sup** [comment]
sure @ashleygwilliams

**Comment by zkat** [comment]
My only suggestion here is that we make a note at the top saying "these are simple bullet-point notes made during the session for those who are curious as to overall points touched. More detailed, specific, and clear notes will be made available soon after watching the recording"

Then this is +1 from me for merging so we can get this PR closed.

**Comment by sup** [comment]
LGTM, merge

**Pull request merged**